RNiTunes/RNiTunes.m

```diff
@@ -196,7 +205,15 @@ @implementation RNiTunes
         NSDictionary *songDictionary = [NSMutableDictionary dictionary];
 
         if (fields == nil) {
-            songDictionary = @{@"albumTitle":albumTitle, @"albumArtist": albumArtist, @"duration":[duration isKindOfClass:[NSString class]] ? [NSNumber numberWithInt:[duration intValue]] : duration, @"genre":genre, @"playCount": [NSNumber numberWithInt:[playCount intValue]], @"title": title};
+            songDictionary = @{
+              @"albumTitle": albumTitle,
+              @"albumArtist": albumArtist,
+              @"assetUrl": assetUrl,
+              @"duration": [duration isKindOfClass:[NSString class]] ? [NSNumber numberWithInt:[duration intValue]] : duration,
+              @"genre": genre,
+              @"playCount": [NSNumber numberWithInt:[playCount intValue]],
+              @"title": title
+            };
         } else {
```

Comment:
Do you need to do anything with `assertUrl` in this case?

Reply:
Yea, way down [here](https://github.com/kadiks/react-native-itunes/blob/087941ce55633baeb54bb33d4e77923715270137/RNiTunes/RNiTunes.m#L386).